share/xtp/xml/neighborlist.xml

```diff
@@ -1,5 +1,5 @@
 <options>
 <neighborlist help="Determines neighbouring pairs for exciton transport" section="sec:neighborlist">
-        <constant  help="cutoff for all other types of pairs" unit="nm" default="1.5">1.5</constant>
+        <constant  help="cutoff for all other types of pairs" unit="nm" default="1.5" choices="float+">1.5</constant>
 </neighborlist>
```

Comment:
there should be a lot more options here.

Reply:
I remember from a chat with @baumeier that there are not *sensible* defaults for this calculator and the user must provide them. But I am not sure

Reply:
Yes, aren't the other options specific for certain molecule types, and hence, the input depends n the naming in the mapping file.